### DISCUSSION TITLE
Is it possible to have Azure security baseline policy in SCA?

### DISCUSSION THREAD

**Discussion by njmulsqb** [wazuh/wazuh#17386] [Q&A]
Hi,

I am moving some servers from Defender for Servers to Wazuh, for hardening I want to have consistent standard being followed on both Azure and Wazuh, azure uses its own security baseline for showing hardening recommendations, is there any way that I could use Azure's policy as SCA in Wazuh?

**Comment by MagicHat101** [comment]
We're in a similar situation but regarding InTune specific CIS benchmarks. We're moving to full Azure Cloud Only and using InTune (no on prem domain). Without having these Intune CIS policies worked into wazuh, the entire SCA portion of wazuh is worthless to us unfortunately.

CIS_Microsoft_Intune_for_Windows_10_Benchmark_v1.1.0.pdf
CIS_Microsoft_Intune_for_Windows_11_Benchmark_v1.0.0.pdf

https://www.cisecurity.org/benchmark/intune

**Reply by njmulsqb** [reply]
When should I expect these rules to come out in Wazuh?

**Comment by Mrgove10** [comment]
+1 on this, also in need of intune benchmark ;)
